tests/unit-tests/__snapshots__/format.test.ts.snap

```diff
@@ -119,8 +119,10 @@ exports[`format > config > default > it formats ../cases/gjs/exported-mod-var.gj
 
 exports[`format > config > default > it formats ../cases/gjs/invalid-template.gjs 1`] = `
 "<template>
+  
               {{#if oops}}
-            </template>
+
```

Comment:
Are we adding newlines on syntax errors now? This was a bug in the past.

Reply:
fixed it. how about now?
```hbs
<template>
  {{#if oops}}
</template>
```

Reply:
This is better, but if our goal is to follow Prettier's existing behavior, it seems that syntax errors should no-op.

Reply:
Oh, okay then

Reply:
fixed